### ISSUE TITLE
Feedback on final project, milestone 2

### ISSUE THREAD

**Issue by barryross** [bryansaylor/homework-portfolio#7]
Hey Bryan, great start to this.  Happy to see you are moving forward with the music blog idea!  I see you have implemented the "read more"/"read less" functionality...great stuff!  Have you thought about any additional jQuery or effects you may want to implement?  How are you feeling about the project overall?

**Comment by bryansaylor** [comment]
There's definitely some more 'toggle' stuff i have planned, and I also want to shrink and lock the video upon scrolling, but after researching it a little it seems pretty involved.  But hopefully i can find a decent solution

**Comment by barryross** [comment]
I think you are referring to having a fixed sidebar or footer or something that would contain the video...is that right? 

Something like this? 

https://css-tricks.com/examples/ScrollingSidebar/

**Comment by bryansaylor** [comment]
Yes, that would be great.  The video is within the article at the top, but
once it is out of view, i would love to do something like that.

**Comment by barryross** [comment]
I think it'd be a cool thing to implement too.  Let me know if you would like a hand with how to approach it!
